src/components/Pill/PillDropdown/PillDropdown.scss

```diff
@@ -15,6 +15,8 @@
 
 .PillDropdown {
     z-index: 500;
+    max-width: 440px;
+    max-height: 440px;
```

Comment:
If you put loads of content in the dropdown, it will be bigger then the max-height. Add a story that covers that case.